packages/components/alert/src/alert.vue

```diff
@@ -5,7 +5,11 @@
       :class="[ns.b(), ns.m(type), ns.is('center', center), ns.is(effect)]"
       role="alert"
     >
-      <el-icon v-if="showIcon && iconComponent" :class="iconClass">
+      <el-icon
+        v-if="showIcon && iconComponent"
+        :class="iconClass"
+        aria-hidden="true"
```

Comment:
Screen readers will not recognise the SVG, so we should add `aria-describedby` and `title` to the SVG.

> - VoiceOver will not recognise the SVG. For VoiceOver there is nothing.
> - NVDA will not recognise the SVG either. For NVDA there is nothing.
> - Narrator will read Image. What kind of image? Not very helpful.
> - Chromevox will not recognise the SVG. For Chromevox there is nothing.
> 
> from [a11y-101.com](https://a11y-101.com/development/svg)

But aria properties can't be easily added to svg icon, because the same icon can have different meanings so the easiest thing to do is to hide it(Standalone, Decorative Icon).

if you have free time, you can read this post(https://css-tricks.com/accessible-svgs/#aa-example-2-standalone-icon-decorative).

Reply:
Thanks for the link.